pylzy/tests/scenarios/env_fail/__init__.py

```diff
@@ -1,31 +0,0 @@
-import tempfile
-import uuid
-
-from lzy.api.v1 import Lzy, op
-
-
-@op
-def main() -> int:
-    return 0
-
-
-WORKFLOW_NAME = "workflow_" + str(uuid.uuid4())
-
-if __name__ == "__main__":
-    with tempfile.NamedTemporaryFile(mode="w", dir="/tmp", suffix=".yaml") as f:
-        f.write(
-            "\n".join(
-                [
-                    "name: py39",
-                    "dependencies:",
-                    "- python==3.9.7",
-                    "- pip",
-                    "- pip:",
-                    "  - kekreader==5653749",
```

Comment:
it is the main string, that we test
let's do a test for a such case

Reply:
Because I deleted the possibility to pass custom yaml, as promised.